tasks/helpers/responsehelper.js

```diff
@@ -0,0 +1,16 @@
+const responseHelper = `
+    /**
+     * @static
+     * @param {response}  response object
+     * @param {message}  message
+     * @param {data}  data
+     * @param {status}  status code
+     * @returns {object} object
+    */
+    export default (res, message, data, status, data) => {
```

Comment:
I think there is a comment left here @hezronkimutai for how we can do this by @its-tayo. 
I will do more of this review to get to a conclusion.

Reply:
`data` is also added twice in this case.